test/langtools/jdk/javadoc/doclet/testNavigation/TestNavigation.java

```diff
@@ -201,57 +172,82 @@ public static class Y extends X {
                 "-sourcepath", src.toString(),
                 "pkg1");
         checkExit(Exit.OK);
 
         checkOrder("pkg1/A.X.html",
-                "Summary",
-                """
-                    <li><a href="#nested-class-summary">Nested</a>&nbsp;|&nbsp;</li>""",
-                """
-                    <li><a href="#field-summary">Field</a>&nbsp;|&nbsp;</li>""",
                 """
-                    <li><a href="#constructor-summary">Constr</a>&nbsp;|&nbsp;</li>""",
-                """
-                    <li><a href="#method-summary">Method</a></li>""");
+                    <ol class="toc-list">
+                    <li><a href="#" tabindex="0">Description</a></li>
+                    <li><a href="#nested-class-summary" tabindex="0">Nested Class Summary</a></li>
+                    <li><a href="#field-summary" tabindex="0">Field Summary</a></li>
+                    <li><a href="#constructor-summary" tabindex="0">Constructor Summary</a></li>
+                    <li><a href="#method-summary" tabindex="0">Method Summary</a></li>
+                    <li><a href="#field-detail" tabindex="0">Field Details</a>
+                    <ol class="toc-list">
+                    <li><a href="#field" tabindex="0">field</a></li>
+                    </ol>
+                    </li>
+                    <li><a href="#constructor-detail" tabindex="0">Constructor Details</a>
+                    <ol class="toc-list">
+                    <li><a href="#%3Cinit%3E()" tabindex="0">X()</a></li>
+                    </ol>
+                    </li>
+                    <li><a href="#method-detail" tabindex="0">Method Details</a>
+                    <ol class="toc-list">
+                    <li><a href="#method()" tabindex="0">method()</a></li>
+                    </ol>
+                    </li>
+                    </ol>""");
 
         checkOrder("pkg1/A.Y.html",
-                "Summary",
-                """
-                    <li><a href="#nested-class-summary">Nested</a>&nbsp;|&nbsp;</li>""",
-                """
-                    <li><a href="#field-summary">Field</a>&nbsp;|&nbsp;</li>""",
                 """
-                    <li><a href="#constructor-summary">Constr</a>&nbsp;|&nbsp;</li>""",
-                """
-                    <li><a href="#method-summary">Method</a></li>""");
+                    <ol class="toc-list">
+                    <li><a href="#" tabindex="0">Description</a></li>
+                    <li><a href="#nested-class-summary" tabindex="0">Nested Class Summary</a></li>
+                    <li><a href="#field-summary" tabindex="0">Field Summary</a></li>
+                    <li><a href="#constructor-summary" tabindex="0">Constructor Summary</a></li>
+                    <li><a href="#method-summary" tabindex="0">Method Summary</a></li>
+                    <li><a href="#constructor-detail" tabindex="0">Constructor Details</a>
+                    <ol class="toc-list">
```

Comment:
General comment: not specific to this instance:
I like that we cleaned up HTML id names in recent releases.